MVC/application/view/components/userPostedData.php

```diff
@@ -0,0 +1,12 @@
+<?php
+  if(isset($_SESSION['userPostedData'])){
```

Comment:
Another alternative way of doing this, which will also reduce the amount of php code used , is to traverse the loop, then close the php tag. Process all the html elements and after processing close the loop. So basically we do not need to process echo every element, instead echo image or content inside html element